src/config/env.ts

```diff
@@ -1,30 +1,48 @@
-export const APP_VERSION = import.meta.env.VITE_VERSION;
-export const DOMAIN = import.meta.env.VITE_GRAASP_DOMAIN;
+const {
+  VITE_VERSION,
+  VITE_GRAASP_API_HOST,
+  VITE_GRAASP_DOMAIN,
+  VITE_GRAASP_ACCOUNT_HOST,
+  VITE_GRAASP_ANALYZER_HOST,
+  VITE_GRAASP_LIBRARY_HOST,
+  VITE_GRAASP_PLAYER_HOST,
+  VITE_GRAASP_AUTH_HOST,
+  VITE_SHOW_NOTIFICATIONS,
+  VITE_GRAASP_ASSETS_URL,
+  VITE_H5P_INTEGRATION_URL,
+  VITE_SENTRY_ENV,
+  VITE_GA_MEASUREMENT_ID,
+  VITE_SENTRY_DSN,
+} =
+  // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+  // @ts-expect-error
```

Comment:
Mhmm you are just missing a `window.d.ts` file that exposes the Cypress key on the global object. We have examples of these in the apps and in auth I think.